accessibility_api/accessibility_lib/scripts/accessible.py

```diff
@@ -0,0 +1,42 @@
+'''
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this file,
+You can obtain one at http://mozilla.org/MPL/2.0/.
+'''
+
+from ctypes import POINTER
+from accessibility_api.accessibility_lib.scripts.constants import (
+    IAccessible_t, IAccessible2_t
+)
+from accessibility_api.accessibility_lib.wrappers.IAccessible import (
+    IAccessible
+)
+from accessibility_api.accessibility_lib.scripts.constants import (
+    ERROR
+)
+
+
+def interface_ptr_types():
+    """
+    Return supported interface pointer types
+    """
+
+    return [POINTER(IAccessible_t), POINTER(IAccessible2_t)]
```

Comment:
nit: each value on the new line (it will be a simpler change to add new things later):
```
[
    POINTER(IAccessible_t), 
    POINTER(IAccessible2_t)
]
```

Also I thought we are getting read of the IA2 stuff, might as well everywhere, including here, why not have that as a separate PR either now or at later time. (maybe keep it in a separate branch, just publish it)